gotty.go

```diff
@@ -21,33 +22,29 @@ import (
 // If something went wrong reading the terminfo database file, an error is
 // returned.
 func OpenTermInfo(termName string) (*TermInfo, error) {
-	var term *TermInfo
-	var err error
 	// Find the environment variables
-	termloc := os.Getenv("TERMINFO")
-	if len(termloc) == 0 {
+	if termloc := os.Getenv("TERMINFO"); len(termloc) > 0 {
+		path := filepath.Join(termloc, string(termName[0]), termName)
```

Comment:
I think using the thing which is semantically correct everywhere is the right thing to do even if it isn't strictly necessary for a given case. Unless there is some massive performance overhead of `filepath` vs `path` but I doubt that.